website/docs/components/rich-tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,125 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="[RT].Toggle" @type="yielded component">
+    The `RichTooltip::Toggle` component (see below).
+  </C.Property>
+  <C.Property @name="[RT].Bubble" @type="yielded component">
+    The `RichTooltip::Bubble` component (see below).
+  </C.Property>
+  <C.Property @name="isOpen" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Controls if the tooltip should be rendered initially opened.
+    <br />
+    <br />
+    _Notice: in this case the tooltip can't be dismissed via `esc` or "click outside" until the end user has interacted with it (it's in a "manual" state)._
+  </C.Property>
+  <C.Property @name="enableSoftEvents" @type="boolean" @default="true" @values={{array "true" "false"}}>
+    Assigns "soft" event listeners (`mouseEnter/Leave` + `focusIn/Out`) to the toggle to control the visibility of the tooltip.
+  </C.Property>
+  <C.Property @name="enableClickEvents" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Assigns a "click" event listener (`onClick`) to the toggle to control the visibility of the tooltip.
+    <br />
+    <br />
+    _Notice: enabling "click" event listeners will automatically disable "soft" event listeners._
```

Comment:
Question: why can't it have both?

Reply:
If both are enabled, the popover can never be opened on click (with a mouse/pointer, I mean) because it already appears on hover. So a "soft+click" combination is practically (almost) identical to a "soft" one. Having only two clear possible interactions makes things easier to explain and more predictable for consumers (and end-users).